s4e-backend/src/test/java/pl/cyfronet/s4e/controller/InvitationControllerTest.java

```diff
@@ -256,25 +241,12 @@ public void shouldReject() throws Exception {
         assertEquals(invitationAfterReject.get().getStatus(), InvitationStatus.REJECTED);
     }
 
-    @Test
-    public void deleteShouldThrowBadRequestOnDifferentUrlInstitutionAndInvitationInstitution() throws Exception {
-        val newInstitution = institutionRepository
-                .save(InvitationHelper.institutionBuilder().build());
-        val invitation = invitationRepository
-                .save(InvitationHelper.invitationBuilder(newInstitution).build());
-        val URL = API_PREFIX_V1 + "/institutions/{institution}/invitations/{token}";
-        mockMvc.perform(delete(URL, institution.getSlug(), invitation.getToken())
-                .contentType(MediaType.APPLICATION_JSON)
-                .with(jwtBearerToken(institutionAdmin, objectMapper))
-        ).andExpect(status().isBadRequest());
-    }
```

Comment:
This test shouldn't be removed, the case of privilage escalation in this point could still be possible, you should just test with an id instead of a token.

Reply:
Show me in which of those lines (in controller) and will agree

```
val invitation = invitationService
                .findByIdAndInstitutionSlug(id, institutionSlug, Invitation.class)
                .orElseThrow(() -> new NotFoundException("Invitation couldn't be found"));

        val deleteInvitationEvent = new OnDeleteInvitationEvent(invitation.getToken(), LocaleContextHolder.getLocale());
        eventPublisher.publishEvent(deleteInvitationEvent);

Reply:
This one could allow for this:
```
@DeleteMapping(value = "/institutions/{institution}/invitations/{id}")
```
The ones you quoted don't have anything to do with that possibility.

Reply:
I can agree only to add a second not found test case when an invitation doesn't exist for certain id. Nothing more nothing less...

Reply:
ok